packages/core/src/constants/provider.ts

```diff
@@ -26,9 +25,13 @@ export const defaultTranscodingProfiles: Array<TranscodingProfile> = [
   },
 ];
 
-export type LivepeerProviderName = 'studio';
-
-export const studio: LivepeerProviderConfig = {
+const _studioConfigConst = {
   name: 'Livepeer Studio',
   baseUrl: 'https://livepeer.studio/api',
+  apiKey: 'a616be3b-8980-4932-8079-0122e0106f95',
 } as const;
+
+export type StudioLivepeerProviderName = typeof _studioConfigConst.name;
```

Comment:
We can probably just remove this export until there's >1 provider. The point of the `LivepeerProviderName` type was to be like `export type LivepeerProviderName = 'studio' | 'catalyst' | 'someotherprovider';`

Reply:
Ahh I see. I can remove this then.